cansav09/sc-rna_seq-reorganization (#28)
AlexsLemonade/training-modules: scRNA-seq/clustering_scRNA-seq.Rmd

---
title: "scRNA-seq_post_processing"
output:   
  html_notebook: 
    toc: true
    toc_float: true
---

**CCDL 2019**

In this notebook, we'll perform dimension reduction and clustering analyses of 
single cell RNA-seq data. 

## Set Up 

> For these analyses, we will need Seurat a lot, so we will go ahead and attach it.  

Reviewer: ```suggestion
For these analyses, we will use the Seurat package.  
```
